lib/fastlane/plugin/emerge/helper/github.rb

```diff
@@ -43,10 +43,7 @@ def self.previous_sha
         if is_push?
           github_event_data.dig(:before)
         else
```

Comment:
Thinking about this more, I think the entire `else` branch here is invalid. If it's a pull request event GitHub will build with a merge commit so it doesn't really make sense to set the previous commit to the previous commit on this branch, because the code being built will also include new changes on the base branch of this PR